WordPressAuthenticator/Sources/Services/LoginFacade.m

```diff
@@ -1,6 +1,6 @@
 #import "LoginFacade.h"
 #import "WordPressXMLRPCAPIFacade.h"
-#import "WPAuthenticator-Swift.h"
+#import <WordPressAuthenticator/WordPressAuthenticator-Swift.h>
```

Comment:
The header above was necessary when the lib was distributed via CocoaPods, but now we can access the Xcode-generated one directly.

This is unrelated with the CP removal for Gutenberg, but I noticed it when grepping "CocoaPods".